source/_integrations/cisco_webex_teams.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: Cisco Webex Teams
```

Comment:
This title is inherited from the integration. Can you link the upstream PR that adjusts this title of the integration upstream?

Thanks! 👍 

../Frenck

Reply:
I haven't changed the upstream name, I was mainly updating the documentation. I'll need to leave the title alone for now then until upstream is changed.